### PR TITLE
ENG-1002: Pull changes from PSMDB Jenkins to template jobs

### PR DESCRIPTION
**Pull request by Sudokamikaze**
These changes were discovered and pulled from Jenkins' Web GUI to have consistent files

### CODE REVIEW

File: psmdb/percona-server-for-mongodb-4.0-template.yml [hunk 1]
```diff
@@ -64,12 +64,12 @@
...
 
         if [ -f /usr/bin/apt-get ]; then
-            sudo apt-get -y install wget
+            sudo DEBIAN_FRONTEND=noninteractive apt-get -y install wget
         else
             sudo yum -y install wget redhat-lsb-core
         fi
 
-        wget https://raw.githubusercontent.com/percona/percona-server-mongodb/${PSM_BRANCH}/percona-packaging/scripts/psmdb_builder.sh
+        wget https://raw.githubusercontent.com/percona/percona-server-mongodb/v4.0/percona-packaging/scripts/psmdb_builder.sh
```
Comment (EvgeniyPatlan) [inline psmdb/percona-server-for-mongodb-4.0-template.yml:72]: no. branch should be a param

File: psmdb/percona-server-for-mongodb-4.0-template.yml [hunk 2]
```diff
@@ -158,12 +158,6 @@
         fi
     - shell: |-
         # run tests
-
-        if [ -f /usr/bin/yum ]; then
-            ulimit -u 64000
-            ulimit -n 64000
-        fi
-
```
Comment (EvgeniyPatlan) [inline psmdb/percona-server-for-mongodb-4.0-template.yml:166]: why? it is needed for correct tests

File: psmdb/percona-server-for-mongodb-4.2-template.yml [hunk 3]
```diff
@@ -64,22 +64,22 @@
...
 
         if [ -f /usr/bin/apt-get ]; then
-            sudo apt-get -y install wget
+            sudo apt-get -y install wget wiredtiger
         else
             sudo yum -y install wget redhat-lsb-core
         fi
 
-        wget https://raw.githubusercontent.com/percona/percona-server-mongodb/${PSM_BRANCH}/percona-packaging/scripts/psmdb_builder.sh
+        wget https://raw.githubusercontent.com/percona/percona-server-mongodb/v4.2/percona-packaging/scripts/psmdb_builder.sh
```
Comment (EvgeniyPatlan) [inline psmdb/percona-server-for-mongodb-4.2-template.yml:72]: branch should be a param

File: psmdb/percona-server-for-mongodb-4.2-template.yml [hunk 4]
```diff
@@ -64,22 +64,22 @@
         set -x
 
         if [ -f /usr/bin/apt-get ]; then
-            sudo apt-get -y install wget
+            sudo apt-get -y install wget wiredtiger
```
Comment (EvgeniyPatlan) [inline psmdb/percona-server-for-mongodb-4.2-template.yml:67]: why?

File: psmdb/percona-server-for-mongodb-4.2-template.yml [hunk 5]
```diff
@@ -208,6 +202,10 @@
 
         DISTRIBUTION="xenial"
 
+        if [ -f /usr/bin/yum ]; then
+            ulimit -u 64000
+            ulimit -n 64000
+        fi
```
Comment (EvgeniyPatlan) [inline psmdb/percona-server-for-mongodb-4.2-template.yml:209]: if distro is xenial and yum?

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: psmdb/percona-server-for-mongodb-4.0-template.yml (hunk 1 above)
Comment (sandraromanchenko) [reply psmdb/percona-server-for-mongodb-4.0-template.yml:72]: yep, Evgeniy is right, branch should be param

File: psmdb/percona-server-for-mongodb-4.0-template.yml (hunk 2 above)
Comment (sandraromanchenko) [reply psmdb/percona-server-for-mongodb-4.0-template.yml:166]: as well, no idea why it's removed

File: psmdb/percona-server-for-mongodb-4.2-template.yml (hunk 3 above)
Comment (sandraromanchenko) [reply psmdb/percona-server-for-mongodb-4.2-template.yml:72]: ++, branch is param

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: psmdb/percona-server-for-mongodb-4.2-template.yml (hunk 4 above)
Comment (sandraromanchenko) [reply psmdb/percona-server-for-mongodb-4.2-template.yml:67]: pls remove for now, I'll add it if required

**Pull request closed without merging**